en/04_Changelogs/5.0.0.md

```diff
@@ -109,6 +109,11 @@ This is a major release and as a result there are a number of breaking API chang
   - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` is now in silverstripe/versioned as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`
   - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` is now in silverstripe/versioned as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`
   - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` is now in silverstripe/versioned as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`
+- Various internal controller methods have been updated to now consistently return an `SilverStripe\Control\HTTPResponse` object, rather than an undeclared `mixed` return type.
+  - These updates do not affect custom controller action methods which can still have an undeclared mixed return type.
+  - The following classes and methods now return a `SilverStripe\Control\HTTPResponse` and will need to be updated if they have been subclassed and overloaded in custom code
```

Comment:
The way this is worded it kinda seems like these two are the _only_ ones being changed (I'm not sure how much this suggestion improves that though)
```suggestion
- Various internal controller methods have been updated to now consistently return an `SilverStripe\Control\HTTPResponse` object, rather than an undeclared `mixed` return type. These updates do not affect custom controller action methods which can still have an undeclared mixed return type.
  - In particular, the following will need to be updated if they have been subclassed and overloaded in custom code:
```

Reply:
Updated